src/components/Common/Components/CartCard/CartCard.jsx

```diff
@@ -0,0 +1,95 @@
+import actionTypes from "../../../../store/actionTypes";
+import "./CartCard.css";
+const CartCard = (props) => {
+  const {
+    _id,
+    name,
+    brand,
+    price,
+    discount,
+    img,
+    qty,
+    updateCartItem,
+    deleteFromCart,
+    addToWishlist,
+    wishlisted,
+    removeFromWishlist,
+  } = props;
+  const image = require(`../../../../assets/images/${img}.png`).default;
```

Comment:
put assets in public folder and you can access with a single / (e.g '/assets/imageName' )